tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -18,34 +18,43 @@ class TestAddNode(ManageTest):
     """
     Automates adding worker nodes to the cluster while IOs
     """
+
     def test_add_node(self):
         """
         Test for adding worker nodes to the cluster while IOs
         """
         dt = config.ENV_DATA['deployment_type']
         if dt == 'ipi':
```

Comment:
you can use this marker @ipi_deployment_required

Reply:
this is not part of his code change?